modules/average_month.py

```diff
@@ -0,0 +1,109 @@
+"""
+This module generates a report which displays average highest and lowest temperature
+as well as average max humidity for a given month
+"""
```

Comment:
Can you confirm from the task details if we need to show avg max humidity or mean humidity?

```suggestion
"""
This module generates a report for the average highest and lowest temperature
as well as average mean humidity for a given month
"""
```

Reply:
its mean humidity. fixed it